.github/workflows/docker-publish.yml

```diff
@@ -1,95 +1,54 @@
 name: Docker
 
-# This workflow uses actions that are not certified by GitHub.
-# They are provided by a third-party and are governed by
-# separate terms of service, privacy policy, and support
-# documentation.
-
 on:
   workflow_dispatch:
   push:
-    branches: [ "master" ]
+    branches: [ "main", "master", "develop" ]
+    # Publish semver tags as releases.
     tags: [ 'v*.*.*' ]
```

Comment:
most of our tags don't start with `v`, this should probably be updated to take `\d+\.\d+\.\d+(-.*)?` tags as well

Note I didn't test that regex, could be wrong

Reply:
I changed it to remove the "v" . 
It looks like the full regex is

^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$

as per chatgpt or

https://semver.org/ has some other long ones. 

Do you want me to use those regexs or is removing V sufficient ?

@MrCreosote

Reply:
Wait, is that actually a regex? Because `*.*.*` isn't valid due to the leading quantifier, and if you remove that `.*.*` matches anything.

Reply:
Ah true, its actually globs. I tried these patterns in this online [tool](https://www.digitalocean.com/community/tools/glob?comments=true&glob=%5B0-9%5D%2A.%5B0-9%5D%2A.%5B0-9%5D%2A&matches=false&tests=%2F%2F%20This%20will%20match%20as%20it%20ends%20with%20%27.js%27&tests=%2Fhello%2Fworld.js&tests=%2F%2F%20This%20won%27t%20match%21&tests=%2Ftest%2Fsome%2Fglobs&tests=1.2.2.3.3.3.3.3.3.3.33.3.3.3.3.%20) 

I added some more tag matches that I think are good enough for this action.